src/globals/scss/styles.scss

```diff
@@ -6,6 +6,7 @@ $css--font-face: false !default;
 $css--helpers: true !default;
 $css--body: true !default;
 $css--reset: false !default;
+$css--use-layer: true !default;
```

Comment:
It might look less "consistent" but I don't think this variable should be using `css` in its name. 
The reason why all the other variables use `css` in their names is because they output selectors and CSS rules.

Maybe we can name this `$global--use-layer`?